docs/config.md

```diff
@@ -81,9 +81,23 @@ labels:
 
 Please refer to the [gRPC guide](./grpc.md).
 
-## (WIP) Justification
+## Justification
 
-TODO
+By default, the client will use `localhost:8080` as the endpoint to retrieve public keys from the JVS.
```

Comment:
I think we should also describe how to use https://github.com/abcxyz/lumberjack/blob/main/clients/go/pkg/justification/processor.go in https://github.com/abcxyz/lumberjack/blob/main/docs/clients.md

Reply:
done